extensions/roc-plugin-dredd/src/actions/dredd.js

```diff
@@ -0,0 +1,33 @@
+import { initLog } from 'roc';
+import Dredd from 'dredd';
+
+const log = initLog();
```

Comment:
Consider adding the `name` and `version` for the extension in the `initLog` function.

https://github.com/rocjs/roc/blob/master/docs/API.md#initlog

This will give more details to the end user about what extension that created the logged message.